commerce_usps.services.yml

```diff
@@ -1 +1,8 @@
 services:
+  commerce_usps.usps_rate_request:
+    class: Drupal\commerce_usps\USPSRateRequest
+    arguments: ['@commerce_usps.usps_shipment']
+
+  commerce_usps.usps_shipment:
+    class: Drupal\commerce_usps\USPSShipment
+    arguments: []
```

Comment:
**CHANGE REQUEST**

If a service does not have any arguments you should simply skip the `arguments` value.